components/tikv_util/src/sys/disk.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
+use fail::fail_point;
+use std::sync::atomic::{AtomicBool, AtomicU64, Ordering};
+pub static DISK_FULL: AtomicBool = AtomicBool::new(false);
+pub static DISK_RESERVED: AtomicU64 = AtomicU64::new(0);
+
+pub fn set_disk_full() {
+    DISK_FULL.store(true, Ordering::Release);
+}
+
+pub fn clear_disk_full() {
+    DISK_FULL.store(false, Ordering::Release);
+}
+
+pub fn is_disk_full() -> bool {
+    DISK_FULL.load(Ordering::Acquire)
+}
+
+//underscore to disable unused warning
+pub fn follower_disk_full_precheck(_store_id: u64) -> bool {
```

Comment:
Should rename to `disk_full_precheck`. And s/disk_full_follower_1/disk_full_store_1